perfscale_regression_ci/kubeburner-object-templates/pod-affinity-anti-affinity-config.yml

```diff
@@ -0,0 +1,49 @@
+jobs:
+  - name: ${S1_PROJ_NAME}
+    namespace: ${S1_PROJ_NAMESPACE}
+    jobIterations: ${S1_PROJ_JOB_ITERATION}
+    qps: ${QPS}
+    burst: ${BURST}
+    namespacedIterations: false
+    podWait: false
```

Comment:
We want to set this to **true** (in all the jobs in this list) so we don't need the wait for running function here